### PR TITLE
Top holders erc721 - Ready for review

### PR DESCRIPTION
**Pull request by henrystats**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [ ] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Comment by jeff-dude** [comment]
do you need full history at this moment, or just noticing the outlier?

**Comment by henrystats** [comment]
ohh ohh I seee. thanks youuu ser... I don't need the full history... so I'll just update the pr as ready to review.

File: models/nft/ethereum/nft_ethereum_top_erc721_holders.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+     wallet_address,
+     nft_contract_address, 
+     balance, 
+     balance/total_supply as supply_share,
+     total_supply, 
+    ROW_NUMBER() OVER (PARTITION BY nft_contract_address ORDER BY balance DESC) as rn
+ FROM 
+ total_supply
+) x 
+-- WHERE rn <= 50
```
Comment (jeff-dude) [inline models/nft/ethereum/nft_ethereum_top_erc721_holders.sql:47]: do you want to keep this to more than 50? then filter on the app instead?

Comment (henrystats) [reply models/nft/ethereum/nft_ethereum_top_erc721_holders.sql:47]: hmm there's already a balances table soo I guess it's better to keep it as top 50. But what would you suggest? happy with either

Comment (henrystats) [reply models/nft/ethereum/nft_ethereum_top_erc721_holders.sql]: updated

**Review by jeff-dude — APPROVED** [review]
thank you ser

**Pull request merged**